app/models/compute_profile.rb

```diff
@@ -20,5 +20,5 @@ class ComputeProfile < ApplicationRecord
   scoped_search :on => :name, :complete_value => true
   default_scope -> { order('compute_profiles.name') }
 
-  scope :visibles, -> { eager_load(:compute_attributes).where('compute_attributes.id > 0') }
+  scope :visibles, -> { joins(:compute_attributes) }
```

Comment:
we don't actually use the eager loaded attributes anywhere, this looks like some weird way to turn a left outer join into an inner join.